src/main/kotlin/pl/edu/prz/kod/domain/Language.kt

```diff
@@ -2,7 +2,8 @@ package pl.edu.prz.kod.domain
 
 enum class Language(val value: String) {
     PYTHON("python"),
-    JAVA("java");
+    JAVA("java"),
+    NODE("node");
```

Comment:
Should it be `node` or `nodejs`? @mateuszaliyev Could You elaborate?

Reply:
> Should it be `node` or `nodejs`? @mateuszaliyev Could You elaborate?

**Node**:
+ Name of the command-line program (`node main.js`)

**Node.js**:
+ `apt-get` package name (`apt-get install nodejs`)
+ Official domain (https://nodejs.org/)
+ [Official trademarked word](https://nodejs.org/static/documents/trademark-policy.pdf)

Personally I think it does not matter as long as it is being used consistently across the codebase. Both names are self-explanatory, especially in the context of this application.

If I had to decide, I would choose Node.js / `nodejs` simply because it is the official name of the runtime.